core/datainput.gms

```diff
@@ -1507,14 +1507,4 @@ $endif.subsectors
 *** initialize global target deviation scalar
 sm_globalBudget_dev = 1;
 
-*** define tolerance level by how much biomass share needs to comply with 2005 historical values
-*** low tolerance for fixed_shares, there is works
-s_histBioShareTolerance = 0.02;
-*** temporary: until subsectors historical FE mix checked -> high tolerance for industry subsectors to make it run 
-$ifthen.subsectors "%industry%" == "subsectors"   !! industry
-s_histBioShareTolerance = 0.3;
```

Comment:
30% of tolerance is quite crazy. I understand why you are doing this, but I am afraid that this is an easy way out, and it will give the wrong impression that things are working meanwhile the issue is not really fixed in the industry subsectors side.